CalibTracker/SiPixelLorentzAngle/python/SiPixelLorentzAnglePCLHarvester_cfi.py

```diff
@@ -3,6 +3,7 @@
 
 SiPixelLorentzAnglePCLHarvester = DQMEDHarvester(
     "SiPixelLorentzAnglePCLHarvester",
+    newmodulelist = cms.string('newmodule.txt'),
```

Comment:
can the list of modules be fully specified in the configuration as a vector of strings instead of reading them from another file?